code/js/modules/Sitelist.js

```diff
@@ -67,6 +67,7 @@
       "bbc": { name: "BBC Radio", url: "http://www.bbc.co.uk/radio", controller: "BBCRadioController.js" },
       "beatport": { name: "Beatport", url: "https://www.beatport.com" },
       "blitzr": { name: "Blitzr", url: "http://www.blitzr.com" },
+      "brainfm": { name: "BrainFM", url: "http://brain.fm" },
```

Comment:
Need to change this line to: `"brain": { name: "BrainFM", url: "http://brain.fm", controller: "BrainFMController.js" },`

The problem is that you included the TLD in your sitename key, which is incorrect (sitename key matches the domain part only, ignoring TLD). This is also why the tests were failing :)